src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -431,543 +443,544 @@ This section guides you step-by-step in setting up an intuitive frontend.
-  CardMedia,
-  MobileStepper,
-  Stack,
-  SwipeableDrawer,
-  TextField,
-  Toolbar,
-  useMediaQuery,
-} from "@mui/material";
-import Card from "@mui/material/Card";
-import CardContent from "@mui/material/CardContent";
-import {
-  AddCircleOutlined,
-  Close,
-  KeyboardArrowLeft,
-  KeyboardArrowRight,
-} from "@mui/icons-material";
-import Paper from "@mui/material/Paper";
-import Typography from "@mui/material/Typography";
-import { useFormik } from "formik";
-import React, { useEffect, useState } from "react";
-import * as yup from "yup";
-import { TZIP21TokenMetadata, UserContext, UserContextType } from "./App";
-import { useSnackbar } from "notistack";
-import { BigNumber } from "bignumber.js";
-import { address, bytes, nat } from "./type-aliases";
-import { char2Bytes } from "@taquito/utils";
-import { TransactionInvalidBeaconError } from "./TransactionInvalidBeaconError";
-```
-
-and some variables inside your `MintPage` Component function
-
-```typescript
-const [activeStep, setActiveStep] = React.useState(0);
-
-const handleNext = () => {
-  setActiveStep((prevActiveStep) => prevActiveStep + 1);
-};
-
-const handleBack = () => {
-  setActiveStep((prevActiveStep) => prevActiveStep - 1);
-};
-
-const handleStepChange = (step: number) => {
-  setActiveStep(step);
-};
-```
+    ```typescript
+    import SwipeableViews from "react-swipeable-views";
+    import OpenWithIcon from "@mui/icons-material/OpenWith";
+    import {
+      Box,
+      Button,
+      CardHeader,
+      CardMedia,
+      MobileStepper,
+      Stack,
+      SwipeableDrawer,
+      TextField,
+      Toolbar,
+      useMediaQuery,
+    } from "@mui/material";
+    import Card from "@mui/material/Card";
+    import CardContent from "@mui/material/CardContent";
+    import {
+      AddCircleOutlined,
+      Close,
+      KeyboardArrowLeft,
+      KeyboardArrowRight,
+    } from "@mui/icons-material";
+    import Paper from "@mui/material/Paper";
+    import Typography from "@mui/material/Typography";
+    import { useFormik } from "formik";
+    import React, { useEffect, useState } from "react";
+    import * as yup from "yup";
+    import { TZIP21TokenMetadata, UserContext, UserContextType } from "./App";
+    import { useSnackbar } from "notistack";
+    import { BigNumber } from "bignumber.js";
+    import { address, bytes, nat } from "./type-aliases";
+    import { char2Bytes } from "@taquito/utils";
+    import { TransactionInvalidBeaconError } from "./TransactionInvalidBeaconError";
+    ```
+
+    and some variables inside your `MintPage` Component function
+
+    ```typescript
+    const [activeStep, setActiveStep] = React.useState(0);
+
+    const handleNext = () => {
+      setActiveStep((prevActiveStep) => prevActiveStep + 1);
+    };
+
+    const handleBack = () => {
+      setActiveStep((prevActiveStep) => prevActiveStep - 1);
+    };
+
+    const handleStepChange = (step: number) => {
+      setActiveStep(step);
+    };
+    ```
 
 ## Let's play
```

Comment:
"Your picture is be pushed to IPFS and displayed." => "Your picture is pushed to IPFS and displayed on the page."